File: 프로그래머스/unrated/181905. 문자열 뒤집기/문자열 뒤집기.py
def solution(my_string, s, e):
    
    if e == 0:
        return my_string
    print(my_string[e:None if s == 0 else s-1:-1], 11)
    return my_string[:s] + my_string[e:None if s == 0 else s-1:-1] + my_string[e+1:]


my_string = "HelloMyMaster"
s, e = 0, 4


print(solution(my_string, s, e))

File: 프로그래머스/unrated/181905. 문자열 뒤집기/문자열 뒤집기.java
class Solution {
    public String solution(String my_string, int s, int e) {
        String first = my_string.substring(0, s);
        String midle = new StringBuffer(my_string.substring(s, e+1)).reverse().toString();
        String last = my_string.substring(e+1, my_string.length());
        // System.out.println(first);
        // System.out.println(midle);
        // System.out.println(last);

        return first + midle + last;
    }
}